sdk/servicebus/azure-servicebus/azure/servicebus/management/_management_client.py

```diff
@@ -0,0 +1,317 @@
+
+        to_update = copy(queue_description._to_internal_entity())  # pylint:disable=protected-access
+
+        for attr in QUEUE_DESCRIPTION_SERIALIZE_ATTRIBUTES:
+            setattr(to_update, attr, getattr(queue_description, attr, None))
+        to_update.default_message_time_to_live = avoid_timedelta_overflow(to_update.default_message_time_to_live)
+        to_update.auto_delete_on_idle = avoid_timedelta_overflow(to_update.auto_delete_on_idle)
+
+        create_entity_body = CreateQueueBody(
+            content=CreateQueueBodyContent(
+                queue_description=to_update,
+            )
+        )
+        request_body = create_entity_body.serialize(is_xml=True)
+        with _handle_response_error():
+            try:
+                et = cast(
+                    ElementTree,
+                    self._impl.queue.put(
+                        queue_description.queue_name,  # type: ignore
+                        request_body,
+                        api_version=constants.API_VERSION,
+                        if_match="*",
+                        **kwargs
+                    )
+                )
+            except ValidationError:
+                # post-hoc try to give a somewhat-justifiable failure reason.
+                raise_with_traceback(
+                    ValueError,
+                    message="queue_description must be a QueueDescription with valid fields, "
+                            "including non-empty string queue name")
+        result = QueueDescription._from_internal_entity(  # pylint:disable=protected-access
+            _convert_xml_to_object(queue_description.queue_name, et)
+        )
+        result.queue_name = queue_description.queue_name
+        return result
+
+    def delete_queue(self, queue_name, **kwargs):
+        # type: (str, Any) -> None
+        """Delete a queue
+
+        :param str queue_name: The name of the queue
+        """
+
+        if not queue_name:
+            raise ValueError("queue_name must not be None or empty")
+        with _handle_response_error():
+            self._impl.queue.delete(queue_name, api_version=constants.API_VERSION, **kwargs)
+
+    def list_queues(self, **kwargs):
+        # type: (Any) -> List[QueueDescription]
+        """List the queues of a ServiceBus namespace
+
+        :keyword int skip: skip this number of queues
+        :keyword int max_count: return at most this number of queues if there are more than this number in
+         the ServiceBus namespace
+        """
+        result = []  # type: List[QueueDescription]
+        internal_queues = self._list_queues(**kwargs)
+        for queue_name, internal_queue in internal_queues:
+            qd = QueueDescription._from_internal_entity(internal_queue)  # pylint:disable=protected-access
+            qd.queue_name = queue_name
+            result.append(qd)
+        return result
+
+    def list_queues_runtime_info(self, **kwargs):
+        # type: (Any) -> List[QueueRuntimeInfo]
```

Comment:
We should be returning Pageable objects for list operations (even if the response isn't pageable).
Infact... the presense of "skip" and "max_count" effectively makes it pageable....
I'm okay if we want to open an issue to address this in the next preview.